Bottom: f4e4c0e8c1fe9b524221d95e47a728895d6d6e72
Top:    fa1c467b4ea1abc4c053add6a494ca856b82e72f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-02-09 11:07:01 -0600

Fix printf in docker_build

Printf statement was expanding a variable into the format string,
which caused problems when that variable contained patterns that
appeared to be invalid format specifiers.


---
diff --git a/ansible-playbook/library/docker_build b/ansible-playbook/library/docker_build
index 8cc11ed..da0c78c 100644
--- a/ansible-playbook/library/docker_build
+++ b/ansible-playbook/library/docker_build
@@ -42,7 +42,7 @@ build_result="${?}"
 
 if [[ ${build_result} != "0" ]]; then
     # build failed
-    printf "{\"failed\": true, \"changed\": true, \"stdout\": \"${build_log}\"}"
+    printf "{\"failed\": true, \"changed\": true, \"stdout\": \"%s\"}" "${build_log}"
     exit 1
 fi
